### PR TITLE
light and sound parameter consistency and fixes

### PR DESCRIPTION
**Pull request by fmessmer**

### CODE REVIEW

**Comment by fmessmer** [comment]
Tested on cob4-1

File: cob_monitoring/src/battery_monitor.py [hunk 1]
```diff
@@ -55,43 +55,184 @@
...
+
+                self.relative_remaining_capacity = self.power_state.relative_remaining_capacity
+                self.set_light(mode, True)
+
 
 if __name__ == "__main__":
-	rospy.init_node("battery_monitor")
-	battery_monitor()
-	rospy.spin()
+    rospy.init_node("battery_light_monitor")
```
Comment (benmaidel) [inline cob_monitoring/src/battery_monitor.py]: battery_monitor instead of battery_light_monitor

Comment (fmessmer) [inline cob_monitoring/src/battery_monitor.py]: jap, good catch...also fixed the class name

**Comment by floweisshardt** [comment]
tested sss part on cob4-1: OK

**Comment by benmaidel** [comment]
FR: OK

**Pull request merged**